gcloud-java-compute/src/main/java/com/google/cloud/compute/Compute.java

```diff
@@ -0,0 +1,2709 @@
+     * {@code value} is interpreted as a regular expression using RE2 syntax. {@code value} must
+     * match the entire field.
+     *
+     * @see <a href="https://github.com/google/re2/wiki/Syntax">RE2</a>
+     */
+    public static RegionFilter equals(RegionField field, String value) {
+      return new RegionFilter(checkNotNull(field), ComparisonOperator.EQ, value);
+    }
+
+    /**
+     * Returns a not-equals filter for the given field and string value. For string fields,
+     * {@code value} is interpreted as a regular expression using RE2 syntax. {@code value} must
+     * match the entire field.
+     *
+     * @see <a href="https://github.com/google/re2/wiki/Syntax">RE2</a>
+     */
+    public static RegionFilter notEquals(RegionField field, String value) {
+      return new RegionFilter(checkNotNull(field), ComparisonOperator.NE, checkNotNull(value));
+    }
+  }
+
+  /**
+   * Class for filtering zone lists.
+   */
+  class ZoneFilter extends ListFilter {
+
+    private static final long serialVersionUID = -3927428278548808737L;
+
+    private ZoneFilter(ZoneField field, ComparisonOperator operator, Object value) {
+      super(field.selector(), operator, value);
+    }
+
+    /**
+     * Returns an equals filter for the given field and string value. For string fields,
+     * {@code value} is interpreted as a regular expression using RE2 syntax. {@code value} must
+     * match the entire field.
+     *
+     * @see <a href="https://github.com/google/re2/wiki/Syntax">RE2</a>
+     */
+    public static ZoneFilter equals(ZoneField field, String value) {
+      return new ZoneFilter(checkNotNull(field), ComparisonOperator.EQ, checkNotNull(value));
+    }
+
+    /**
+     * Returns a not-equals filter for the given field and string value. For string fields,
+     * {@code value} is interpreted as a regular expression using RE2 syntax. {@code value} must
+     * match the entire field.
+     *
+     * @see <a href="https://github.com/google/re2/wiki/Syntax">RE2</a>
+     */
+    public static ZoneFilter notEquals(ZoneField field, String value) {
+      return new ZoneFilter(checkNotNull(field), ComparisonOperator.NE, checkNotNull(value));
+    }
+  }
+
+  /**
+   * Class for filtering operation lists.
+   */
+  class OperationFilter extends ListFilter {
+
+    private static final long serialVersionUID = -3202249202748346427L;
+
+    private OperationFilter(OperationField field, ComparisonOperator operator, Object value) {
+      super(field.selector(), operator, value);
+    }
+
+    /**
+     * Returns an equals filter for the given field and string value. For string fields,
+     * {@code value} is interpreted as a regular expression using RE2 syntax. {@code value} must
+     * match the entire field.
+     *
+     * @see <a href="https://github.com/google/re2/wiki/Syntax">RE2</a>
+     */
+    public static OperationFilter equals(OperationField field, String value) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The method name and parameter number are suspiciously close to equals(Object)](https://www.codacy.com/app/mziccard/gcloud-java/file/2740139423/issues/source?bid=3123104&fileBranchId=3354337#l724)